kong/plugins/aws-lambda/README.md

```diff
@@ -0,0 +1,41 @@
+Working:
+- Add plugin to api
+- Specify aws credentials (IAM access_key and secret_key) in config
+- Specify region, function name, body in config
+- Return response value
+- Return appropriate error response on request if api.upstream_url is *not* aws-lambda://
+- Allow merging of query parameters from api to lambda payload
+- Error handling
+
+ToDo:
+- Allow merging of body from api to lambda
+- Allow passing body from client through to lambda (without specifying in config)
```

Comment:
This is actually done - had just missed the README update. New commit with relevant README updates coming momentarily.